<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintainceFaultQueryPage</class>
 <widget class="QWidget" name="MaintainceFaultQueryPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
		background-color:black;
}
QPushButton
{
	font: 20px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(0, 128, 128);
	border-top: 2px solid white;	
	border-left: 2px solid white;
border-radius:8px;
}
QLabel
{
	color:white;
	font: 20px, &quot;微软雅黑&quot;;

}</string>
  </property>
  <widget class="QPushButton" name="TC1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>TC1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>车辆</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="MP1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>MP1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="M1">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>M1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="M2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>M2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MP2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>120</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>MP2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TC2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>120</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>TC2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>610</y>
     <width>1018</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(0, 0, 0);

	border:1px solid white;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="returnBtn">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>620</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 20px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(0, 128, 0);
    border-top: 2px solid white;
	border-left: 2px solid white;

}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>系统</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="TCMS">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>TCMS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BCU">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>制动系统</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DCU">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>牵引系统</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ACM">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>210</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>辅助系统</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EDCU">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>210</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>车门系统</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PIS">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>210</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>广播系统</string>
   </property>
  </widget>
  <widget class="QPushButton" name="WMS">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>细水雾</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HVAC">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>280</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>空调系统</string>
   </property>
  </widget>
  <widget class="QLabel" name="mp2wheel2lbl">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>80</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
border-radius:7px;
border:2px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="mp1wheel1lbl">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
border-radius:7px;
border:2px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tc1wheel1lbl">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
border-radius:7px;
border:2px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="mp1wheel2lbl">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
border-radius:7px;
border:2px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="m2wheel1lbl">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>80</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
border-radius:7px;
border:2px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="mp1lbl">
   <property name="geometry">
    <rect>
     <x>314</x>
     <y>35</y>
     <width>100</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font:20px &quot;微软雅黑&quot;;
	border:2px solid white;
	background-color: rgb(255,255,153);
color:black;
</string>
   </property>
   <property name="text">
    <string>1002</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tc2wheel2lbl">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>80</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
border-radius:7px;
border:2px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="m2lbl">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>35</y>
     <width>100</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font:20px &quot;微软雅黑&quot;;
	border:2px solid white;
	background-color: rgb(255,255,153);
color:black;
</string>
   </property>
   <property name="text">
    <string>1004</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tc1keylbl">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>35</y>
     <width>31</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/TC1traingraphKeynoact.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tc2lbl">
   <property name="geometry">
    <rect>
     <x>726</x>
     <y>35</y>
     <width>100</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font:20px &quot;微软雅黑&quot;;
	border:2px solid white;
	background-color: rgb(255,255,153);
color:black;
</string>
   </property>
   <property name="text">
    <string>1006</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="m1lbl">
   <property name="geometry">
    <rect>
     <x>417</x>
     <y>35</y>
     <width>100</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font:20px &quot;微软雅黑&quot;;
	border:2px solid white;
	background-color: rgb(255,255,153);
color:black;
</string>
   </property>
   <property name="text">
    <string>1003</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="mp2wheel1lbl">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>80</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
border-radius:7px;
border:2px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="m1wheel2lbl">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>80</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
border-radius:7px;
border:2px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tc2wheel1lbl">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>80</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
border-radius:7px;
border:2px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tc2keylbl">
   <property name="geometry">
    <rect>
     <x>825</x>
     <y>35</y>
     <width>31</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/TC2traingraphKeynoact.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="mp2lbl">
   <property name="geometry">
    <rect>
     <x>623</x>
     <y>35</y>
     <width>100</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font:20px &quot;微软雅黑&quot;;
	border:2px solid white;
	background-color: rgb(255,255,153);
color:black;
</string>
   </property>
   <property name="text">
    <string>1005</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tc1wheel2lbl">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
border-radius:7px;
border:2px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tc1lbl">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>35</y>
     <width>100</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font:20px &quot;微软雅黑&quot;;
	border:2px solid white;
	background-color: rgb(255,255,153);
color:black;
</string>
   </property>
   <property name="text">
    <string>1001</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="m1wheel1lbl">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
border-radius:7px;
border:2px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="m2wheel2lbl">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>80</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
border-radius:7px;
border:2px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="TCMSFaultBtn_2">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>460</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>460</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>记录</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>520</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>未记录</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="TCMSFaultBtn_3">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>520</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: gray;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
